dev/NavigationView/readme.md

```diff
@@ -0,0 +1,56 @@
+# NavigationView Control
+
+NavigationView is a powerful control makes it easy to integrate navigation into apps.
```

Comment:
suggestion: NavigationView is a powerful control that makes it easy to integrate a rich navigation experience into apps.

Reply:
Good idea, updated now.